packages/react-error-overlay/package.json

```diff
@@ -39,28 +39,29 @@
     "babel-jest": "^22.1.0",
     "babel-loader": "^8.0.0-beta.0",
     "babel-preset-react-app": "^3.1.1",
-    "chalk": "^2.1.0",
-    "chokidar": "^2.0.0",
+    "chalk": "^2.3.2",
+    "chokidar": "^2.0.2",
     "cross-env": "5.1.3",
-    "eslint": "4.15.0",
+    "eslint": "4.19.0",
     "eslint-config-react-app": "^2.1.0",
-    "eslint-plugin-flowtype": "2.41.0",
-    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-flowtype": "2.46.1",
+    "eslint-plugin-import": "2.9.0",
     "eslint-plugin-jsx-a11y": "6.0.3",
-    "eslint-plugin-react": "7.5.1",
-    "flow-bin": "^0.63.1",
+    "eslint-plugin-react": "7.7.0",
+    "flow-bin": "^0.66.0",
     "html-entities": "1.2.1",
-    "jest": "22.1.2",
-    "jest-fetch-mock": "1.2.1",
+    "jest": "22.4.2",
+    "jest-fetch-mock": "1.4.2",
     "object-assign": "4.1.1",
     "promise": "8.0.1",
     "raw-loader": "^0.5.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "rimraf": "^2.6.1",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "rimraf": "^2.6.2",
     "settle-promise": "1.0.0",
-    "source-map": "0.5.6",
-    "webpack": "^3.6.0"
+    "source-map": "0.7.2",
```

Comment:
Reverting this to `0.5.6` should fix most of the CI errors. Some other CI issues have been fixed on `next` so please merge that into this branch as well.

Reply:
@iansu Merge or **rebase**?

Reply:
rebase.